### PR TITLE
Adds fetching for  a better gasPrice estimate from eth gas station

### PR DESCRIPTION
**Pull request by lsaether**
Description in the title. Depends on new version of eac.js-lib.

### CODE REVIEW

File: app/stores/TransactionStore.js [hunk 1]
```diff
@@ -427,6 +427,16 @@ export class TransactionStore {
...
+  async getAvgGasPrice() {
+    const ethGasStationStream = await fetch('https://ethgasstation.info/json/ethgasAPI.json');
+    const avgGasPrice = (await ethGasStationStream.json()).average;
+    const gwei = 1000000000;
+    // EthGasStation returns the average gas in such a weird format,
+    // where the number XX represents GWEI with one decimal place. So,
+    // the return value "33" would mean 3.3 gwei, hence this strange return.
+    return (avgGasPrice * gwei) / 10;
+  }
+
```
Comment (kosecki123) [inline app/stores/TransactionStore.js]: This implementation is inferior to the one we have in timenode-core https://github.com/ethereum-alarm-clock/timenode-core/pull/295/files#diff-b10e04079822a0a088ccb4d434a520cfR87 `public async getAdvancedNetworkGasPrice(): Promise<GasPriceEstimation>`

**Comment by lsaether** [comment]
Depends on the new release of `timenode-core` which exposes the function `getAdvancedNetwrokGasPrice()`

File: app/stores/TransactionStore.js [hunk 2]
```diff
@@ -49,6 +51,7 @@ export class TransactionStore {
     this._features = featuresService;
     this._helper = helper;
     this._bucketHelper = bucketHelper;
+    this.util = new TNUtil(this._web3);
```
Comment (Kuzirashi) [inline app/stores/TransactionStore.js]: You've declared `_util` above, but here you set `util`. Please stick to one property name.

Comment (lsaether) [reply app/stores/TransactionStore.js]: Fixed

**Pull request merged**